### ISSUE TITLE
services list does not work

### ISSUE THREAD

**Issue by SebastianHGonzalez** [SebastianHGonzalez/python-kong-client#28]
``` python
>>> list(KongAdminClient(<kong-url>).services.list)

Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "/home/sgonzalez/PycharmProjects/python-kong-gateaway/kong/kong_clients.py", line 178, in generator
    offset, cached, _ = self._send_list(size, offset, **query_params)
  File "/home/sgonzalez/PycharmProjects/python-kong-gateaway/kong/kong_clients.py", line 132, in _send_list
    return offset, elements, response['total']
KeyError: 'total'
```

**Comment by SebastianHGonzalez** [comment]
kong server 0.13.0 does not return total although documentation said so

**Comment by SebastianHGonzalez** [comment]
reported Kong/kong#3372

**Comment by SebastianHGonzalez** [comment]
solved in #29
